.github/workflows/TestTugboat.yml

```diff
@@ -49,7 +51,7 @@ jobs:
       # preview - they should be the same.
       - name: Test Generated Files
         run: |
-          cmp -b drainpipe/.tugboat/config.yml .tugboat/config.yml
```

Comment:
I was trying to find a way to test my changes by generating a more robust config file and testing it against a more complete example.  The test runner doesn't seem to be able to locate this file, though, so I'm not sure if I'm doing something wrong or if there's a better way to test my work.

Reply:
@KeyboardCowboy the way we're currently testing it is by having the Tugboat file generated match the one we're using to create a Tugboat environment - bit of a hack, but unfortunately we can't generate a tugboat configuration on the fly. If you add elastic search to `.tugboat/config.yml` then that should be fine. Is there anything we can add to verify that elasticsearch has started correctly and is usable by Drupal?